docs/guides/databases_with_concord.md

```diff
@@ -2,14 +2,134 @@
+    case CONNECTION_MADE:
+        /* Successfully connected, you can proceed using database functions. */
+        log_trace("[libpq] Successfully connected to the postgres database server.");
+        break;
+    case CONNECTION_STARTED:
+        /* Still connecting, but really soon you will be connected. */
+        log_trace("[libpq] Waiting for connection to be made.");
+        break;
+    case CONNECTION_AWAITING_RESPONSE:
+        /* Waiting for a response from the database, probably will respond soon, if properly configured. */
+        log_trace("[libpq] Waiting for a response from the server.");
+        break;
+    case CONNECTION_AUTH_OK:
+        /* Authentication was successful, but the backend is still starting up. */
+        log_trace("[libpq] Received authentication; waiting for backend start-up to finish.");
+        break;
+    case CONNECTION_SSL_STARTUP:
+        /* Negotiating SSL encryption, since the connected database was using SSL encryption. */
+        log_trace("[libpq] Almost connecting; negotiating SSL encryption.");
+        break;
+    case CONNECTION_SETENV:
+        /* Negotiating environment-driven parameter settings, soonly will be connected */
+        log_trace("[libpq] Almost connecting; negotiating environment-driven parameter settings.");
+        break;
+    default:
+        /* Ugh, something went wrong while connecting to the database, probably a wrong password, firewall, or something blocking the connection. */
+        log_fatal("[libpq] Error when trying to connect to the postgres database server. [%s]\n", PQerrorMessage(conn));
+        PQfinish(conn);
+}
+```
+
+### Creating a table
+
+We got the database connected, so now we can now proceed creating a table, since it's required before saving any data into it.
+
+```c
+
+/* Here we are using PQexec to execute commands, on this example, we are using "CREATE TABLE" command, which creates a table in the database, but with the `IF NOT EXISTS` clause, it will only create the table if it doesn't exist, and the table will have two collumns with the value INT. */
+PGresult *res = PQexec(conn, "CREATE TABLE IF NOT EXISTS concord_guides(user_id INT, guild_id INT);");
+
+if (PQresultStatus(res) != PGRES_COMMAND_OK) {
+    /* Something went wrong while creating the table, so we are going to log the error message of what went wrong. */
+    log_fatal("[libpq] Error when trying to create the table. [%s]", PQerrorMessage(conn));
+    PQclear(res);
+    /* Even not properly being executed, this isn't, at least most of the time, the fault of the connection, so we are not closing it. */
+    // PQfinish(conn);
+    return;
+} else {
+    log_debug("[libpq] Successfully created the table.");
+}
+```
+
+### Saving data into the table
+
+Now that we have a table, we can now proceed saving data into it, which is really easy to do, since it's almost equal to create a table, the only difference is the command we send to the database to be executed.
```

Comment:
Now that we have a table, we can now proceed saving data into it. It's really easy to do, since it's almost the same as creating a table. The only difference is the command we send to the database to be executed.